app/views/course/leaderboards/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header
+= page_header(@settings.title)
```

Comment:
Correct, I forgot about that.  But need to use `format_inline_text(@settings.title)` to remove dangerous tags.